docs/Python-API.rst

```diff
@@ -12,6 +12,7 @@ Data Structure API
     Dataset
     Booster
     CVBooster
+    Sequence
 
 Training API
```

Comment:
Please add `Sequence` here
https://github.com/microsoft/LightGBM/blob/c7134fa7ccbe1a1b1819ac2a66407504d3fab27a/python-package/lightgbm/basic.py#L1944

Reply:
Let's add this in follow-up PR.